commands/preview.js

```diff
@@ -24,52 +26,65 @@
  * @api assertions
  */
 
-
 var path = require('path');
-var site = require(path.join(path.resolve('./'), '/tests/system/site.json'));
-var qs = require('querystring');
 
-site = site.profiles[site.activeProfile];
+try {
+    var site = require(path.join(path.resolve('./'), '/tests/system/site.json'));
+} catch (e) {
+    if (e instanceof Error && e.code === 'MODULE_NOT_FOUND') {
+        console.log('Not using optional site.json...');
+    }
+}
+var qs = require('querystring');
 
 exports.command = function(url, callback) {
     var browser = this;
 
-    if (typeof url === 'function') {
-        callback = url;
-        url = site.siteUrl;
-    }
+    if (site) { 
+        site = site.profiles[site.activeProfile];
 
-    // First checks for the URL, otherwise uses the site.siteURL, then makes sure
-    // that there is an http prefix. The preveiw function doesn't need this, but
-    // the browser.get() method does.
-    url = url || site.siteUrl;
+        if (typeof url === 'function') {
+            callback = url;
+            url = site.siteUrl;
+        }
 
-    if (!url.match(/^http/)) {
-        throw new Error('Site URL must be correctly formatted');
-    }
+        // First checks for the URL, otherwise uses the site.siteURL, then makes sure
+        // that there is an http prefix. The preview function doesn't need this, but
+        // the browser.get() method does.
+        url = url || site.siteUrl;
 
-    // If the production flag is set, just runs a `get()` on the URL.
-    if (site.production) {
-        return browser.get(url, function(result) {
-            if (typeof callback === 'function') {
-                callback.call(browser, result);
-            }
-        });
-    }
+        if (!url.match(/^http/)) {
+            throw new Error('Site URL must be correctly formatted');
+        }
 
-    var bundleUrl = site.bundleUrl || 'http://localhost:8080';
+        // If the production flag is set, just runs a `get()` on the URL.
+        if (site.production) {
+            return browser.get(url, function(result) {
+                if (typeof callback === 'function') {
+                    callback.call(browser, result);
+                }
+            });
+        }
 
-    var params = qs.stringify({'url': url, 'site_folder': bundleUrl});
+        var bundleUrl = site.bundleUrl || 'http://localhost:8080';
```

Comment:
I would suggest to default to the secure version: `https://localhost:8443/adaptive.js`